src/cloud-api-adaptor/podvm/Makefile.inc

```diff
@@ -135,6 +135,7 @@ $(KATA_AGENT): $(FORCE_TARGET) | $(STATIC_LIBSECCOMP)
 	install -D --compare "$(KATA_AGENT_SRC)/target/$(RUST_TARGET)/$(KATA_AGENT_BUILD_TYPE)/$(@F)" "$@"
 
 $(STATIC_LIBSECCOMP): $(KATA_CONTAINERS_SRC)
+	INSTALL_IN_GOPATH=false $(KATA_CONTAINERS_SRC)/ci/install_yq.sh
```

Comment:
@stevenhorsman `src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder` installs yq 4.35.1 whereas kata's 3.4.1. Will it overwrite 4.35.1 one?

Yeah, we talked about yq on peer-pods many times but my brain cleaned up these completely.

Reply:
It will, but Magnus wrote https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/hack/yq-shim.sh so we can handle both versions 😃

Reply:
oh, cool!